modules/widgets-panel/assets/js/editor/favorites-component.js

```diff
@@ -0,0 +1,140 @@
+import ComponentBase from 'elementor-api/modules/component-base';
+import * as commandsData from './commands-data/';
+import * as commands from './commands';
+import ignoreSectionWidgets from './helpers/ignoreSectionWidgets';
+
+export default class FavoritesComponent extends ComponentBase {
+	getNamespace() {
+		return 'panel-favorites';
+	}
+
+	defaultCommands() {
+		return this.importCommands( commands );
+	}
+
+	defaultData() {
+		return this.importCommands( commandsData );
+	}
+
+	registerAPI() {
+		super.registerAPI();
+		$e.routes.on( 'run:after', ( component, route ) => {
+			if ( 'panel/elements/categories' === route ) {
+				this.menuId = 'e-favorite-widget-context-menu';
+				this.favoriteWidgetMenu = null;
+				this.clickedWidget = null;
+
+				this.addFavoriteWidgetsEvents();
+				this.addToolTip();
+			}
+		} );
+	}
+
+	addToolTip() {
+		const favoriteCategory = jQuery( '#elementor-panel-category-favorites .elementor-panel-category-title > i' );
+		if ( favoriteCategory ) {
```

Comment:
It's always true because jQuery always returns an object. use  `$favoriteCategory.length`.